### PR TITLE
Add TiKV debug service

### PR DESCRIPTION
**Pull request by overvenus**
Sometime we want to debug without stop TiKV, so here comes Debug service!

### CODE REVIEW

File: proto/debugpb.proto [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    uint64 region_id = 1;
+}
+
+message RegionInfoResponse {
+    message Info {
+        raft_serverpb.RaftLocalState raft_local_state = 1;
+        raft_serverpb.RaftApplyState raft_apply_state = 2;
+    }
+
+    repeated Info info = 1;
```
Comment (siddontang) [inline proto/debugpb.proto]: we may also need to know in the region

1. approximate size
2. approximate keys
3. approximate rows
4. ... 

we can use table user property to get these /cc @huachaohuang

File: proto/debugpb.proto [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+message RegionInfoResponse {
+    raft_serverpb.RaftLocalState raft_local_state = 1;
+    raft_serverpb.RaftApplyState raft_apply_state = 2;
+}
+
+message SizeRequest {
+    // If region_id is 0, then calcate all region's size.
+    uint64 region_id = 1;
+    // If there is no cf provided, then calcate all cf.
+    repeated string cfs = 2;
```
Comment (hicqu) [inline proto/debugpb.proto]: Could `cfs` be a enum?

Comment (hicqu) [reply proto/debugpb.proto]: We have 4 constants represent those cfs. They are `&'static str`. Got it.

**Comment by hicqu** [comment]
LGTM.

**Comment by nolouch** [comment]
LGTM

**Pull request merged**